webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7063,7 +7063,7 @@ var userOverride = {
                 isAgentOffline
             ) {
                 var userSession = KommunicateUtils.isSessionStorageAvailable()
-                    ? JSON.parse(sessionStorage.kommunicate)
+                    ? (sessionStorage.kommunicate ? JSON.parse(sessionStorage.kommunicate) : {})
```

Comment:
can we move the new check to outer level? like combine both the checks as it seems similar